### DISCUSSION TITLE
Disabling buildkit while performing bentoml containerize

### DISCUSSION THREAD

**Discussion by sagarjoglekar** [bentoml/BentoML#3036] [Q&A]
Hi There, 
I was wondering if there is a way to disable buildkit while doing the containerize step? Defaulting to multi-arch makes using bentoml a bit tricky with CI tools. 
Thanks!

**Comment by aarnphm** [comment]
You can disable buildkit with `DOCKER_BUILDKIT=0`

**Reply by sagarjoglekar** [reply]
Thanks a lot for the response! I tried exporting this as an env variable and also supplying it inline like so `DOCKER_BUILDKIT=0 docker build ... `, but it always fails with `Error: [cli] `containerize` failed: BentoML requires Docker Buildx to be installed to support multi-arch builds. ...`

**Reply by sagarjoglekar** [reply]
Hi @aarnphm ,
Thanks a lot for helping. I am not a contributor to the project but I was just trying to understand the containerize step. I came across this https://github.com/bentoml/BentoML/blob/945189daadba9f04c690b22540c71022f0646a0a/bentoml/bentos.py#L487
where the DOCKER_BUILDKIT env var is hardcoded to 1. And as far as I can tell, it is not updated before being passed to the subprocess. Could this be the reason why the exported env var is not working? 
Thanks again!
